web/src/compose_recipient.js

```diff
@@ -148,7 +148,8 @@ function switch_message_type(message_type) {
     };
     update_compose_for_message_type(message_type, opts);
     update_placeholder_text();
-    compose_ui.set_focus(message_type, opts);
+    opts.message_type = message_type;
+    compose_ui.set_focus(opts);
```

Comment:
I don't understand this one -- isn't `message_type` already in there?

Reply:
Oh woops, yup! Dang we have some messy types around this stuff.